### ISSUE TITLE
FEATURE - Add a Line Chart to `OdometerView`

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#353]
## Describe the feature you want the app to have that it currently doesn't
* Add a line graph on the top half of the screen of the `OdometerView` 
   * Date on the X axis, and odometer readings on the Y axis
* The graph should have different series for the different vehicles
   * Later on depending on if #350 is done before this one, changing the filter, will change which vehicle is shown on the graph

## Describe alternatives you've considered
* N/A, open to any ideas!

## Additional context
* This repo may be helpful for making the graph!

https://github.com/jordibruin/Swift-Charts-Examples

The graph could look something like this, with a legend on it with the vehicle name to label them

![Screenshot 2024-10-23 at 2 28 22 PM](https://github.com/user-attachments/assets/59f8b4ca-4c3e-4467-ae39-d3cd65e2bb9b)

**Comment by lukasskolek** [comment]
Hi, can I take this issue? 🙂

**Comment by MobinEzzati** [comment]
Hi, can I work on it ?

**Comment by mikaelacaron** [comment]
@MobinEzzati no, it's already been assigned, and there's an open PR that I'll be looking at soon that resolves this

please look for another issue thanks!
also sorry for the late reply, I didn't see this notification somehow

**Comment by lukasskolek** [comment]
Hello, If Mikaela is okay with it Id be happy to work on this with you, I
am super new to this.

Best Lukas

LUKAS SKOLEK
***@***.***
+421 903 276 627
Kadnarova 106
831 06 Bratislava
Slovakia

**Comment by mikaelacaron** [comment]
@lukasskolek there's nothing left for you to do! I still need to review the PR you've already submitted, after that it'll be accepted or you'll see a list of things to fix in order for the PR to be merged

also see on GitHub you responded via email, and it added a bunch of info about your email, you may want to go into GitHub in your commend and erase that...
